### PR TITLE
[Docs] Druid 29.0.0 release notes

### PR DESCRIPTION
**Pull request by ektravel**
Release and upgrade notes for Druid 29.0.0

### CODE REVIEW

**Review by 317brian — CHANGES_REQUESTED** [review]
Some minor nits and questions

**Comment by 317brian** [comment]
We also need to add this as experimetnal in highlights @ektravel https://github.com/apache/druid/pull/15689

File: docs/release-info/release-notes.md [hunk 1]
```diff
@@ -57,50 +57,557 @@ For tips about how to write a good release note, see [Release notes](https://git
...
+The **Load query detail archive** now supports loading queries by selecting a JSON file directly or dragging the file into the dialog.
+
+![Load query detail archive](./assets/image02.png)
+
+[#15632](https://github.com/apache/druid/pull/15632)
+
+#### Improved time chart brush and added auto-granularity
+
+* Added the notion of timezone in the explore view.
+* Added `chronoshift` as a dependency.
```
[the remark this answers is not in the document]
Comment (ektravel) [reply docs/release-info/release-notes.md:184]: Moved line 184 to "Dependency updates"

File: docs/release-info/release-notes.md [hunk 2]
```diff
@@ -57,50 +57,557 @@ For tips about how to write a good release note, see [Release notes](https://git
...
+* Time chart is now able to automatically pick a granularity if "auto" is selected (which is the default) based on the current time filter extent.
+* Brush is now automatically enabled in the time chart.
+* Brush interval snaps to the selected time granularity.
+* Added a highlight bubble to all visualizations (except table because it has its own).
+
+[#14990](https://github.com/apache/druid/pull/14990)
+
+#### Toggle to fail on empty inserts
+
+Added a new toggle to fail when an ingestion query produces no data.
```
[the remark this answers is not in the document]
Comment (ektravel) [reply docs/release-info/release-notes.md:194]: Updated

File: docs/release-info/release-notes.md [hunk 3]
```diff
@@ -57,50 +57,557 @@ For tips about how to write a good release note, see [Release notes](https://git
...
+#### Segment allocation improvements
+
+Improved segment allocation as follows:
+
+* Enhanced polling in segment allocation queue [#15590](https://github.com/apache/druid/pull/15590)
+* Fixed an issue in segment allocation that could cause loss of appended data when running interleaved append and replace tasks [#15459](https://github.com/apache/druid/pull/15459)
+
+#### Other ingestion improvements
+
+* Added a context parameter `useConcurrentLocks` for concurrent locks. You can set it for an individual task or at a cluster level using `druid.indexer.task.default.context` [#15684](https://github.com/apache/druid/pull/15684)
```
[the remark this answers is not in the document]
Comment (ektravel) [reply docs/release-info/release-notes.md:244]: Good catch. Added to Highlights, Upgrade notes, and upgrade-notes.md

File: docs/release-info/release-notes.md [hunk 4]
```diff
@@ -57,50 +57,557 @@ For tips about how to write a good release note, see [Release notes](https://git
...
+[#15543](https://github.com/apache/druid/pull/15543)
+
+#### Improved lookups
+
+Enhanced lookups as follows:
+
+* Improved loading and dropping of containers for lookups to reduce inconsistencies during updates [#14806](https://github.com/apache/druid/pull/14806)
+* Changed behavior for initialization of lookups to load the first lookup as is, regardless of cache status [#15598](https://github.com/apache/druid/pull/15598)
+
+#### Enabled query request queuing by default when total laning is turned on
```
[the remark this answers is not in the document]
Comment (ektravel) [reply docs/release-info/release-notes.md:381]: Yes, I think it should. Added to "Upgrade notes" and upgrade-notes.md

**Comment by ektravel** [comment]
I'll add https://github.com/apache/druid/pull/15689 once it gets merged.

File: docs/release-info/release-notes.md [hunk 5]
```diff
@@ -57,50 +57,609 @@ For tips about how to write a good release note, see [Release notes](https://git
...
+#### Added system fields to input sources
+
+Added the option to return system fields when defining an input source. This allows for ingestion of metadata, such as an S3 object's URI.
+
+[#15276](https://github.com/apache/druid/pull/15276)
+
+#### Changed how Druid allocates weekly segments
+
+When the requested granularity is a month or larger but a segment can't be allocated, Druid resorts to day partitioning.
+Unless explicitly specified, Druid skips week-granularity segments for data partitioning because these segments don't align with the end of the month or more coarse-grained intervals.
```
Comment (cryptoe) [inline docs/release-info/release-notes.md:269]: I feel for a user this release not does not add value. I think we should mention the old behavior and the new behavior and what should the person do if he want's the old behavior. 
The reasoning of why can go into the PR description. 

cc @kfaraz @AmatyaAvadhanula

Comment (kfaraz) [reply docs/release-info/release-notes.md:269]: @cryptoe , there is no way to get back to the old behaviour now.
Old behaviour was that if allocation is not possible for month, we try week next.
In the new behaviour, we will try skip trying week and try day directly.

In the new behaviour, week segments can only be allocated if the chosen partitioning in the append task is WEEK.

File: docs/release-info/release-notes.md [hunk 6]
```diff
@@ -57,50 +57,609 @@ For tips about how to write a good release note, see [Release notes](https://git
...
+
+### General ingestion
+
+#### Added system fields to input sources
+
+Added the option to return system fields when defining an input source. This allows for ingestion of metadata, such as an S3 object's URI.
+
+[#15276](https://github.com/apache/druid/pull/15276)
+
+#### Changed how Druid allocates weekly segments
```
Comment (kfaraz) [inline docs/release-info/release-notes.md:266]: This entry should also be added as a bullet in `Segment allocation improvements section`.

**Review by 317brian — APPROVED** [review]
LGTM

**Pull request merged**